internal/faustjs.org/docs/blocks/gutenberg-blocks.mdx

```diff
@@ -0,0 +1,55 @@
+---
+slug: gutenberg-blocks-provider
+title: Getting Started with Gutenberg Blocks Provider
+---
+
+# Getting Started with Gutenberg Blocks Provider
+
+## Quick Start
+
+:::note
+
+Make sure you have completed the initial setup for Faust at [Getting Started](/docs/getting-started).
+
+:::
+
+Install the blocks:
+
+```bash
+npm i @faustwp/blocks
+```
+
+Open ```_app.js``` and import the blocks provider:
+
+```js
+import { WordPressBlocksProvider } from '@faustwp/blocks';
+```
+
+Then, inside your templates you need to pass on the ```contentBlocks``` data in your ```WordPressBlocksViewer```:
+
+```js
+import { WordPressBlocksViewer } from '@faustwp/blocks';
+
+const { title, content, featuredImage, date, author, contentBlocks } =
+    props.data.post;
+const blocks = flatListToHierarchical(contentBlocks);
+
+<ContentWrapper>
+   <WordPressBlocksViewer contentBlocks={blocks} />
+</ContentWrapper>
+```
+
+Example ```contentBlock``` GraphQL query fragment:
+```
+contentBlocks(flat: true) {
+  __typename
+  renderedHtml
+  id: nodeId
+  parentId
+  ${fragmentData(components).keys}
```

Comment:
`fragmentData` is not provided in the example. Try to pass on the Fragment directly:
```
${components.CoreParagraph.fragments.entry}
contentBlocks(flat: true) {
  __typename
  renderedHtml
  id: nodeId
  parentId
  ...on CoreParagraphFragment
}
```